index.bs

```diff
@@ -850,14 +850,14 @@ To <dfn>populate the pose</dfn> of an {{XRSpace}} |space| in an {{XRSpace}} |bas
   1. Let |transform| be |pose|'s {{XRPose/transform}}.
   1. Query the [=/XR device=]'s tracking system for |space|'s pose relative to |baseSpace| at the time represented by |frame|, then perform the following steps:
     <dl class="switch">
-      <dt> If the tracking system provides a [=6DoF=] pose:
+      <dt> If the tracking system provides an actively tracked or statically known [=6DoF=] pose for |space|'s pose relative to |baseSpace|:
         <dd> Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
         <dd> Set |transform|'s {{XRRigidTransform/position}} to the position of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
         <dd> Set |pose|'s {{XRPose/emulatedPosition}} to <code>false</code>.
 
-      <dt> Else if the tracking system provides a [=3DoF=] pose or indicates that the pose position is emulated:
+      <dt> Else if the tracking system provides a [=3DoF=] pose or a [=6DoF=] pose whose position is estimated for |space|'s pose relative to |baseSpace|:
```

Comment:
nit: What about...
><dt> Else if the tracking system provides a [=3DoF=] pose or a [=6DoF=] pose whose position is estimated and/or not actively tracked for |space|'s pose relative to |baseSpace|:

Reply:
One catch with that phrasing is a "statically known" transform (e.g. grip space to target ray space) might be interpreted as "not actively tracked".

I went with this:

> Else if the tracking system provides a [=3DoF=] pose or a [=6DoF=] pose whose position is neither actively tracked nor statically known for |space|'s pose relative to |baseSpace|:

This lost the word "estimated", but maybe being more specific here is better.  We can mop up any remaining concerns here when we resolve #534.